bcda/web/api.go

```diff
@@ -32,8 +32,12 @@ import (
 
 var qc *que.Client
 
+const (
+	groupAll = "all"
+)
+
 /*
-	swagger:route GET /api/v1/Patient/$export bulkData bulkPatientRequest
+	swagger:route GET /api/v1/Group/[id]/$export bulkData bulkGroupRequest
```

Comment:
I was talking to @epfrankle this morning; we should not make Swagger changes until the User Guide is updated.  This will allow us to release `/Group` without a dependency on the User Guide.
Can you add details to the User Guide ticket that updating Swagger should be part of the A/C?

Reply:
Yep thought about that, also the metadata endpoint should be part of that AC too. Its already added in the ticket, but I should have removed the actual comments here...good eyes.

Reply:
Should we add some details/commentary in the Swagger description for `/Group` regarding the use of the default group ID (i.e., `all`)?  Otherwise, the users won't know how to call the endpoint.
Should the group ID be passed into Swagger in a fashion similar to the job number?